backend/dataall/modules/datasets/handlers/glue_table_handler.py

```diff
@@ -18,7 +17,7 @@ class DatasetColumnGlueHandler:
     @Worker.handler(path='glue.dataset.database.tables')
     def list_tables(engine, task: models.Task):
```

Comment:
This method has a very bad name, not a problem of modularization but the function is actually syncing tables from glue catalog to data.all. And the handler should be called something like glue_table_sync_handler